lib/rmagick_internal.rb

```diff
@@ -1213,6 +1213,16 @@ def each
         end
       end # class Magick::Image::View::Rows
     end # class Magick::Image::View
+
+    private
+
+    def opaque_value
+      Magick.const_defined?('OpaqueOpacity') ? OpaqueOpacity : OpaqueAlpha
```

Comment:
If this depends on version, maybe it would be good to check that here instead:

```
Magick::Magick_version =~ /ImageMagick (7\.\d+\.\d+)-(\d+) / ? OpaqueOpacity : OpaqueAlpha
```

That will make it a little more clear *why* we have the conditional, and when we can remove it.
(Really think we should probably have a method to return the version.)

Reply:
I don't think we should use a regex every time that opaque_value is being used but I agree that a version check might be better here. Will see if I can change this somehow.

Reply:
I just created a new PR that adds `Magick::Magick_lib_version` to make version checking easier in #567. After that has been merged (if you are okay with this addition) I can update this PR to use that.